packages/headless/src/api/commerce/search/query-suggest/query-suggest-request.ts

```diff
@@ -0,0 +1,41 @@
+import {BaseParam} from '../../../platform-service-params';
+import {
+  ClientIdParam,
+  CurrencyParam,
+  LanguageParam,
+  QueryParam,
+  TrackingIdParam,
+  ContextParam,
+  CountryParam,
+} from '../../commerce-api-params';
+import {baseRequest} from '../../common/request';
+
+export type QuerySuggestRequest = BaseParam &
+  TrackingIdParam &
```

Comment:
There is a set of mandatory repeating params. Perhaps we need to define a specific type for it?